src/store/reducers/describe.ts

```diff
@@ -28,21 +27,34 @@ const describe: Reducer<
             };
         }
         case FETCH_DESCRIBE.SUCCESS: {
-            let newData;
+            const data = action.data;
 
-            if (action.data.Path) {
+            const isCurrentDescribePath = data.Path === state.currentDescribePath;
+
+            // Do not reset loading state if it is not the current describe path
+            // May be needed in case of schema path change while previous fetch has not been finished
+            const wasLoaded = isCurrentDescribePath || state.wasLoaded;
+            const loading = !wasLoaded;
```

Comment:
it seems these values are independent. When you start a request - `loading` is always true. If you want not to render loader in any cases, you should check it in component.
Or you may write a selector like `selectIsLoadingNewSchema`.

Reply:
Changed it to ` const loading = isCurrentDescribePath ? false : state.loading`